docs/application/dotnet/guides/personal/authentication.md

```diff
@@ -1,17 +1,13 @@
 # Authentication and Authorization
```

Comment:
If you think this file, `authentication.md`, should be reused someday, you can leave this file.
Otherwise, please remove this file.

Reply:
In the future, there are chances of redesigning of oauth 2.0. Then we may need to reuse this file again. lets keep this file for now.